homeassistant/components/rachio/strings.json

```diff
@@ -31,6 +31,9 @@
     "binary_sensor": {
       "rain": {
         "name": "Rain"
+      },
+      "flow": {
+        "name": "Flow"
```

Comment:
Does this entity name sound logical? Since the device class is problem so it would mean that there is no flow when the state is `on` right?

Reply:
I was struggling with how to name this too. The frontend and automation editor either show OK or Problem so I was going with the flow being in one of those states for the name because it was easier to understand. Open to suggestions however.